### PR TITLE
Configurable liveness probe initial delays for etcd and kube-apiserver in GCE

### PR DESCRIPTION
**Pull request by wojtek-t**
**Release note**:
```release-note
Configurable liveness probe initial delays for etcd and kube-apiserver in GCE
```

### CODE REVIEW

File: cluster/saltbase/salt/etcd/etcd.manifest [hunk 1]
```diff
@@ -22,6 +22,7 @@
 {% if pillar.get('storage_backend', 'etcd3') == 'etcd3' -%}
   {% set quota_bytes = '--quota-backend-bytes=4294967296' -%}
 {% endif -%}
+{% set liveness_probe_initial_delay = pillar.get('kube_apiserver_liveness_probe_initial_delay', 15) -%}
```
Comment (mborsz) [inline cluster/saltbase/salt/etcd/etcd.manifest]: etcd_liveness_probe_initial_delay?

Comment (wojtek-t) [reply cluster/saltbase/salt/etcd/etcd.manifest]: done

**Pull request merged**